sdk/python/tests/test_bigquery_ingestion.py

```diff
@@ -0,0 +1,149 @@
+import random
+import time
+from datetime import datetime, timedelta
+
+import pandas as pd
+import pytest
+from google.cloud import bigquery
+
+from feast.data_source import BigQuerySource
+from feast.feature import Feature
+from feast.feature_store import FeatureStore
+from feast.feature_view import FeatureView
+from feast.protos.feast.types.EntityKey_pb2 import EntityKey as EntityKeyProto
+from feast.protos.feast.types.Value_pb2 import Value as ValueProto
+from feast.repo_config import LocalOnlineStoreConfig, OnlineStoreConfig, RepoConfig
+from feast.value_type import ValueType
+
+
+@pytest.mark.integration
+class TestMaterializeFromBigQueryToDatastore:
+    def setup_method(self):
+        self.client = bigquery.Client()
+        self.gcp_project = self.client.project
+        self.bigquery_dataset = "test_ingestion"
```

Comment:
We should make a new dataset every time. Have a look here: https://github.com/feast-dev/feast/blob/master/sdk/python/tests/test_historical_retrieval.py#L381